test/integration/scene3d_config.cc

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) 2017 Open Source Robotics Foundation
```

Comment:
```suggestion
 * Copyright (C) 2022 Open Source Robotics Foundation
```

Reply:
I'm going to keep the date the same, since it's just a piece of the former file.